aws_lambda_powertools/logging/logger.py

```diff
@@ -167,12 +169,16 @@ def _init_logger(self, **kwargs):
         self._logger.addHandler(self._handler)
         self.structure_logs(**kwargs)
 
-        logger.debug("Adding filter in root logger to suppress child logger records to bubble up")
-        for handler in logging.root.handlers:
-            # It'll add a filter to suppress any child logger from self.service
-            # Where service is Order, it'll reject parent logger Order,
-            # and child loggers such as Order.checkout, Order.shared
-            handler.addFilter(SuppressFilter(self.service))
+        # Pytest Live Log feature duplicates log records for colored output
+        # but we explicitly add a filter for log deduplication.
+        # This flag disables this protection when you explicit want logs to be duplicated (#262)
+        if not self._is_deduplication_disabled:
+            logger.debug("Adding filter in root logger to suppress child logger records to bubble up")
+            for handler in logging.root.handlers:
+                # It'll add a filter to suppress any child logger from self.service
+                # Where service is Order, it'll reject parent logger Order,
+                # and child loggers such as Order.checkout, Order.shared
```

Comment:
__NITPICK__: This seems to be an example but it's not marked explicitly as such.